neurom/fst/_neuronfunc.py

```diff
@@ -33,7 +33,12 @@
 from neurom.core.types import NeuriteType
 from neurom.core.types import tree_type_checker as is_type
 from neurom.core.dataformat import COLS
-from neurom import morphmath as mm
+from neurom import morphmath
+
+
+def neurons_pop(nrns):
```

Comment:
I don't feel 'pop' is a correct name here. I sounds to me that the result should be removed from nrns.
neurons_as_iterable ? - not super pretty though -